python/publish-layers.sh

```diff
@@ -90,51 +92,51 @@ function publish-python38-x86 {
     done
 }
 
-function build-python39-arm64 {
```

Comment:
It looks like the python39 build functions were replaced by the new python310. Can you add back the python39 ones as well?

Reply:
copy pasting. corrected. thank you

Reply:
@umaannamalai any other updates needed?

Reply:
Looks good - we will merge. Thanks for contributing!